### ISSUE TITLE
base64 on Android returns image path

### ISSUE THREAD

**Issue by felipemillhouse** [Shobbak/react-native-compressor#45]
There is some issue on android method to return base64 image and not the image path. On iOS it works like a charm. I was using `"react-native-compressor": "^1.0.0"` I've updated it to `1.0.3` but no luck with it

```typescript
for (const image of images) {
    const compressed = await Image.compress(image.uri, {
      returnableOutputType: 'base64',
      compressionMethod: 'auto',
    })
  
    compressedImages.push({
      image: compressed,
      height: image.height,
      width: image.width,
    })
  }
  console.log(compressedImages)
...
```
return:
```typescript
[{
  height: 2612,
  image: "file:///data/user/0/com.myapp/cache/e013a0a3-773c-4155-9001-746b456407a0.jpg",
  width: 1958
]}
```

It happens only with `compressionMethod: 'auto'`

**Comment by nomi9995** [comment]
@felipemillhouse 

just released `1.1.0` version of the compressor which has fixed this issue.

```
yarn add react-native-compressor@1.1.0
```

**Comment by felipemillhouse** [comment]
@nomi9995 I just tested it and it works 10/10. Just one more thing if you want to fix that, I noticed and I forgot to tell you. On android, using manual `compressionMethod` the base64 value comes with white spaces, and in my validation process ( using `class-validator` lib on backend) it says `invalid base64 value`. So I have to remove those white spaces (`_.replace(compressed, /\s/g, '')`) before send it to my backend.

Again... it just happens on android, and I didn't test it after 1.1.0 compressor version, I decided to keep the regex on my code just to be safer.

Congrats for that lib, it's pretty cool and easy to use it.
